packages/react-aria-components/src/Menu.tsx

```diff
@@ -233,7 +209,6 @@ function MenuInner<T extends object>({props, collection, menuRef: ref}: MenuInne
             scrollRef={ref} />
         </Provider>
       </div>
-      <div ref={setPopoverContainer} style={{width: '100vw', position: 'absolute', top: 0, ...leftOffset}} />
```

Comment:
The above focus scope changes mean we no longer need to mount the submenu inside the parent menu. This wasn't possible when the menu is embedded inside a larger popover because it would get clipped by overflow: auto. `data-react-aria-top-layer` prevents the parent overflow from making the submenu `aria-hidden`.